projects/interlinked_intricate_network/index.md

```diff
@@ -0,0 +1,218 @@
+
+*Definition Fractals*
+A fractal is a never-ending pattern. Fractals are infinitely complex patterns that are self-similar across different scales. They are created by repeating a simple process over and over in an ongoing feedback loop. Driven by recursion, fractals are images of dynamic systems – the pictures of Chaos. Geometrically, they exist in between our familiar dimensions. Fractal patterns are extremely familiar, since nature is full of fractals. For instance: trees, rivers, coastlines, mountains, clouds, seashells, hurricanes, etc. Abstract fractals – such as the Mandelbrot Set – can be generated by a computer calculating a simple equation over and over.
+
+![example_fractals](http://fractalfoundation.wolfesongs.com/wp-content/uploads/2009/01/sierpinski-zoom41.gif)
+
+[Source: Website Fractal Foundation](http://fractalfoundation.org/resources/what-are-fractals/)
+ 
+
+### execution
+
+The people taking part only got the sheet of paper with the description of the algorithm, no additional tools or materials. The first step of the algorithm asks them to get the relevant material, needed for executing the code, so that the materials picked are comparable but not the same for all the people.  
+ 
+>   **you need:**
+>* three text/ painting tools 
+>   1. thick and dark
+>   2. less thick and dark than 1.
+>   3. less thick and dark than 2.
+>
+>* One work surfaces DIN A4 or bigger
+>
+>**the algorithm**
+>1. Subdivide your work surface into three parts, so that each part got at least two >connections to the edge of the work surface.
+>2. Mark the two of them that are not connected with text/ painting tool no. 1
+>3. Draw three triangles with text/ painting tool no. 3 within these areas
+>4. Read the following step completely before executing.
+>5. Draw another triangle in the middle of each side, outside of the triangles 
+>6. The sides of the new triangles are smaller than the side of the triangle you attached them
+>7. One of the sides of the new triangles is a part of the side of the triangle it is attached to (The new triangle is attached outside of the old triangle)
+>8. On the same principle draw further triangles at the sides of the triangle from step 5 that are lying freely. 
+>9. Repeat this for all the new triangles until the new triangles are smaller than the stroke width of your text/ paint tool.
+>10. Read the following step completely before executing
+>11. Draw straight lines with text/ paint tool no 2., that are each on the one hand side connected with the corners of the triangles drawn in step 3 and on the other hand side with the free corner of the triangle drawn in step 3 facing the other one.
+>12. Continue drawing the lines until you reach the edge of the work surface.
+
+![](./assets/images/002_normal.png)
+ 
+### constant parameters
+ 
+One work surface<br />
```

Comment:
if you want a breakline in markdown just add to whitespaces at the end of the line